runtime/centrifuge/src/lib.rs

```diff
@@ -418,6 +429,92 @@ impl orml_asset_registry::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	pub LiquidityPoolsPalletIndex: PalletIndex = <LiquidityPools as PalletInfoAccess>::index() as u8;
```

Comment:
On the development runtime, this is used in the genesis spec to register assets. This won't be needed here since we are going through governance for that right?

Reply:
Nuno wanted to have the registration as part of the chain spec for local chains. Which I think makes sense. Maybe lets leave a comment here for the future ready to use that then.